parent/pom.xml

```diff
@@ -346,6 +343,12 @@
             <artifactId>scalatest_${scala.compat.version}</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
where this one is use in the code ? I am missing something I think

Reply:
Do you mean jsr305? It's an optional library that contain annotations that one can use in the code to assist automated bug detection tools. We don't really use it (hence scope provided), but it is used in many other libraries that we use, and without in being available at compile time Scala compiler generates warnings. So, adding it as a globally provided library for the project is merely a measure to minimize the amount of false positive warning during the build process.